packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -137,14 +137,25 @@ html`<main>${bodyText}</main>`
 
 Expressions in the child position can take many kinds of values:
 
-* Primitive values likes strings, numbers, and booleans
+* Primitive values
+* Non-rendering values
```

Comment:
I think this organization is mixing the _type_ of value with the _result_ of certain values. I would probably organize this list purely by type:

- Primitives
- `TemplateResult` objects created with the `html` and `svg` functions
- DOM Nodes
- The special sentinels `nothing` and `noChange`
- Arrays or iterables of any of the above

Reply:
This makes sense to me. To reconcile the two orgs, I might add a sentence after the list that says something like:

A few values get special treatment. These include `null`, `undefined`, the empty string, and the sentinel values `nothing` and `noChange`. For more information, see [Non-rendering values](link) and [Signaling no change](link?).

Reply:
Actually, stepping back a bit, adding a bullet for "the special sentinels `nothing` and `noChange`" means we have to add another section in here just to explain that there are two sentinel values that do different things and are discussed elsewhere. That seems not great.

I think I'd lean towards either dropping this bullet & adding a paragraph like I suggested above, or keeping Steve's original list and just resigning ourselves that this list buckets values by treatment, rather than by type.

Reply:
Attempted to address feedback, ptal.